### PR TITLE
Added menu handler

### PR DESCRIPTION
**Pull request by johnTurknett**
Added JS handler for mobile menu control.

### CODE REVIEW

File: src/components/store/layout/header.js [hunk 1]
```diff
@@ -9,34 +9,28 @@ import useCart from "../../../hooks/cart";
...
-              role="button"
-              className="navbar-burger burger"
-              aria-label="menu"
-              aria-expanded="false"
-            >
-              <span aria-hidden="true"></span>
-              <span aria-hidden="true"></span>
-              <span aria-hidden="true"></span>
-            </a>
+        <div className="level">
```
Comment (widox) [inline src/components/store/layout/header.js:20]: wow. this is next level stuff! 🥁

**Comment by widox** [comment]
Just a thought, not sure what the goal is...but after clicking the menu, can it close after a link is chosen or the page changes?

**Comment by johnTurknett** [comment]
> Just a thought, not sure what the goal is...but after clicking the menu, can it close after a link is chosen or the page changes?

Yeah, valid point. I will see if the react router has a callback so that we don't need to implement that logic on every menu route.

**Comment by johnTurknett** [comment]
@widox I added closing the menu on selection.

**Review by widox — APPROVED** [review]

**Pull request merged**